lib/msnodesqlv8/connection-pool.js

```diff
@@ -58,6 +58,7 @@ class ConnectionPool extends BaseConnectionPool {
         }
 
         IDS.add(tds, 'Connection', connedtionId)
+        tds.setUseUTC(this.config.useUTC)
```

Comment:
```suggestion
        tds.setUseUTC(this.config.options.useUTC)
```

Reply:
I think this should actually be under options if we want to use what's already in the docs.

Reply:
good spot!